### PR TITLE
Changes for 1.9.0 rc2

### PR DESCRIPTION
**Pull request by ajazam**
## Urgency
- [ ] Blocker 
- [ ] High
- [x] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

**Comment by lingmann** [comment]
@joel-hamill I don't think I can review this... we need someone who has used GCE and these Ansible scripts. Not sure who that is currently, perhaps someone from support or whoever has been doing scale testing on GCE?

File: 1.9/administration/installing/cloud/gce.md [hunk 1]
```diff
@@ -145,7 +145,7 @@ You must create the RSA public/private keypairs to allow passwordless logins via
     $ cd dcos-gce
     ```
 
-1.  Review and customize the `dcos_gce/group_vars/all`. You should review `project`, `subnet`, `login_name`, `bootstrap_public_ip`, and `zone`. To install DC/OS v1.9-dev ensure dcos_installer_download_path = "https://downloads.dcos.io/dcos/testing/master/{{ dcos_Installer_download_path }}?_ga=1.104771143.674486911.1475259096"
+1.  Review and customize the `dcos_gce/group_vars/all`. You should review `project`, `subnet`, `login_name`, `bootstrap_public_ip`, and `zone`. To install DC/OS v1.9.0 rc2 ensure dcos_installer_download_path = "https://downloads.dcos.io/dcos/EarlyAccess/commit/7f1ce42734aa54053291f403d71e3cb378bd13f3/dcos_generate_config.sh?_ga=1.103056512.1648812748.1479632870"
```
Comment (karlkfi) [inline 1.9/administration/installing/cloud/gce.md]: Remove the `_ga` url param. It's just for google analytics, mapping to a website session.

Comment (karlkfi) [inline 1.9/administration/installing/cloud/gce.md]: Also note: 1.9.0-rc3 should be out today and 1.9.0 early next week.

**Comment by ajazam** [comment]
I can see rc3 yet. Ill wait for this, remove the ga reference and update PR or create new PR.

**Comment by joel-hamill** [comment]
@ajazam okay, so can this PR be closed?

**Pull request closed without merging**